package.json

```diff
@@ -81,7 +85,8 @@
     "url": "https://github.com/stimulus-use/stimulus-use"
   },
   "peerDependencies": {
-    "stimulus": ">=1.1.1 <3"
+    "@hotwired/stimulus": ">=3",
+    "stimulus": ">=3"
```

Comment:
Why to add peerDependency "stimulus": ">=3"? 

If the user has installed "@hotwired/stimulus" then a warning will raise up:

`warning " > stimulus-use@0.50.0-2" has unmet peer dependency "stimulus@>=3".`

Reply:
Since quite some time passed in the meantime I removed the explicit `peerDependencies` on `stimulus`